src/components/FrameworkGrid/FrameworkGrid.tsx

```diff
@@ -71,9 +71,10 @@ const frameworks = [
 
 interface FrameworkGridProps {
   currentKey?: Platform;
+  isGen1?: boolean;
 }
 
-export const FrameworkGrid = ({ currentKey }: FrameworkGridProps) => {
+export const FrameworkGrid = ({ currentKey, isGen1 }: FrameworkGridProps) => {
```

Comment:
For these changes I added a `isGen1` attribute. Is this okay or would it be better to just have each component handle gen1 by looking at the url?

Reply:
Maybe we should have a shared `isGen1` util that checks the path, and then that is shared between the different components?